packages/commons-test/src/riskAnalysisTestUtils.ts

```diff
@@ -73,6 +127,32 @@ export const validatedRiskAnalysis3_0_Pa: RiskAnalysisValidatedForm = {
   ],
 };
 
+export const validRiskAnalysis1_0_Private: RiskAnalysisFormToValidate = {
```

Comment:
Should this also be called "expired" as done for 2.0 above?

```suggestion
export const expiredRiskAnalysis1_0_Private: RiskAnalysisFormToValidate = {
```